### PR TITLE
Add 2021 firmware for Lupus 12126 and 12127

### PR DESCRIPTION
**Pull request by cromefire**
Both can be found together in a firmware zip on their pages:
- https://www.lupus-electronics.de/shop/Alarm-Smarthome/Smarthome-Hausautomation/LUPUS-1-Kanal-Relais-mit-ZigBee-Repeater-p.html
- https://www.lupus-electronics.de/shop/Alarm-Smarthome/Smarthome-Hausautomation/LUPUS-2-Kanal-Relais-mit-ZigBee-Repeater-p.html

### CODE REVIEW

**Comment by cromefire** [comment]
Hmm apparently they share the same ids whcih makes them override each other...
```console
$ node scripts/add.js [...]/PRS3CH1_00.00.05.11TC.zigbee
Updated existing entry ({"fileVersion":511,"fileSize":201260,"manufacturerCode":10132,"imageType":1025,"sha512":"76a327ab014803b6f18376343b1f75de474e7e5ad1b6a0e276668052362e6aff2bccf44bf92683217228cd4049fa1b273231284f4a25aac7858b363f2b09e05b","url":"https://github.com/Koenkk/zigbee-OTA/raw/master/images/ClimaxTechnology/PRS3CH1_00.00.05.11TC.zigbee","path":"images/ClimaxTechnology/PRS3CH1_00.00.05.11TC.zigbee"})
$ node scripts/add.js [...]/PRS3CH2_00.00.05.12TC.zigbee
Updated existing entry ({"fileVersion":512,"fileSize":204164,"manufacturerCode":10132,"imageType":1025,"sha512":"904609129ff2445cc36e0a01033d584797bd784a2033defc2dd1b2f269e9318b6f6bd31f8b1d07e308c4281243bfbde72219de347b6c23e3a1fd4300bfe17aa9","url":"https://github.com/Koenkk/zigbee-OTA/raw/master/images/ClimaxTechnology/PRS3CH2_00.00.05.12TC.zigbee","path":"images/ClimaxTechnology/PRS3CH2_00.00.05.12TC.zigbee"})
```

Any way that can be fixed?

**Comment by cromefire** [comment]
Welp as I don't have the 12126 to test or anything anyway, I just removed for now

**Comment by cromefire** [comment]
Okay I think I got it, at least if it's similar to other devices

File: index.json [hunk 1]
```diff
@@ -3740,5 +3740,25 @@
...
+        "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/ClimaxTechnology/PRS3CH1_00.00.05.11TC.zigbee",
+        "path": "images/ClimaxTechnology/PRS3CH1_00.00.05.11TC.zigbee"
+    },
+    {
+        "fileVersion": 512,
+        "fileSize": 204164,
+        "manufacturerCode": 10132,
+        "imageType": 1025,
+        "sha512": "904609129ff2445cc36e0a01033d584797bd784a2033defc2dd1b2f269e9318b6f6bd31f8b1d07e308c4281243bfbde72219de347b6c23e3a1fd4300bfe17aa9",
+        "modelId": "12127",
```
Comment (Koenkk) [inline index.json:3760]: This should be `PRS3CH2_00.00.05.12TC` (it should be the zigbee model ID)

File: index.json [hunk 2]
```diff
@@ -3740,5 +3740,25 @@
...
         "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Legrand/1021-0034-00064203-NLY.zigbee",
         "path": "images/Legrand/1021-0034-00064203-NLY.zigbee"
+    },
+    {
+        "fileVersion": 511,
+        "fileSize": 201260,
+        "manufacturerCode": 10132,
+        "imageType": 1025,
+        "sha512": "76a327ab014803b6f18376343b1f75de474e7e5ad1b6a0e276668052362e6aff2bccf44bf92683217228cd4049fa1b273231284f4a25aac7858b363f2b09e05b",
+        "modelId": "12126",
```
Comment (Koenkk) [inline index.json:3750]:
```suggestion
        "modelId": "PRS3CH1_00.00.05.11TC",
```

File: index.json (hunk 1 above)
Comment (cromefire) [reply index.json:3760]: In z2m it shows up as 12127 in the OTA view or do I have to look somewhere else?

Comment (Koenkk) [reply index.json:3760]: Looks at the `Zigbee model` in the device about page

Comment (cromefire) [reply index.json:3760]: I'll look it up once I'm back home in the evening.

Comment (cromefire) [reply index.json:3760]: Yeah you're right, it's `PRS3CH2_00.00.05.12TC`. I'll fix it tomorrow (or in my time zone later today strictly speaking...).

**Comment by cromefire** [comment]
That should do it.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**